src/components/ha-areas-display-editor.ts

```diff
@@ -33,8 +33,11 @@ export class HaAreasDisplayEditor extends LitElement {
 
   @property({ type: Boolean }) public required = false;
 
+  @property({ type: Boolean, attribute: "show-navigation-button" })
+  public showNavigationButton = false;
+
   protected render(): TemplateResult {
-    const compare = areaCompare(this.hass.areas, this.value?.order);
+    const compare = areaCompare(this.hass.areas);
```

Comment:
Why dont we sort them by the order anymore?

Reply:
The items-display-editor handles the sorting and the filtering. We only need the default list with the default order here.